lldb/source/DataFormatters/FormatManager.cpp

```diff
@@ -72,6 +72,7 @@ static constexpr FormatInfo g_format_infos[] = {
     {eFormatInstruction, 'i', "instruction"},
     {eFormatVoid, 'v', "void"},
     {eFormatUnicode8, 'u', "unicode8"},
+    {eFormatCheriotCapability, '\0', "cheriot capability"},
```

Comment:
What does `'`0'` do here?

Reply:
Could you make it `'C'` or something?

Reply:
It provides a single-character name for the format. `\0` is the "none" sentinel. Unfortunately 'c' and 'C' are both taken already.

Reply:
I could do something more exotic like '@'?